contracts/cw1-subkeys/src/state.rs

```diff
@@ -33,38 +33,13 @@ const PREFIX_PERMISSIONS: &[u8] = b"permissions";
 
 /// returns a bucket with all permissions (query by subkey)
 pub fn permissions<S: Storage>(storage: &mut S) -> Bucket<S, Permissions> {
-    bucket(PREFIX_PERMISSIONS, storage)
+    bucket(storage, PREFIX_PERMISSIONS)
```

Comment:
I'm about to change this all again :wink: 
https://github.com/CosmWasm/cosmwasm/pull/559

Reply:
:-)